src/features/pages/acuity/acuity-feature-guide/data.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { AcuityFeaturesType } from '../../../components/molecules/image-text-boxes/types'
+
+const deriv_acuity_row_download = 'https://deriv.link/3Vk37Ds'
```

Comment:
can these url be imported from contants?

Reply:
yes we can , previously @nikita-deriv  added a comment on this ,  to move to this folder  since we are not using these links anywhere else

Reply:
yes move it to constants.tsx file

Reply:
sure